dinersclub/reloadly.go

```diff
@@ -74,6 +59,43 @@ func reloadlyErrorResult(res *Result, err error, details *json.RawMessage) (*Res
 	return res, err
 }
 
+func (p *ReloadlyProvider) getCredentials(pool *pgxpool.Pool, pageid string) (*Credentials, error) {
+	query := `SELECT details FROM credentials WHERE facebook_page_id=$1 LIMIT 1`
+	var c Credentials
+	row := pool.QueryRow(context.Background(), query, pageid)
+	err := row.Scan(&c.Details)
+
+	if err == pgx.ErrNoRows {
+		return nil, nil
+	}
+
+	return &c, err
+}
+
+func (p *ReloadlyProvider) Auth(pool *pgxpool.Pool, event *PaymentEvent) error {
+	crds, err := p.getCredentials(pool, event.Pageid)
+	if err != nil {
+		return err
+	}
+	if crds == nil {
+		msg := fmt.Sprintf(`No credentials were found for pageid: %s`, event.Pageid)
```

Comment:
you can use https://pkg.go.dev/fmt#Errorf which is just syntactic sugar for this.